<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>theFile</string>
  </property>
  <property name="windowIcon">
   <iconset theme="thefile" resource="resources.qrc">
    <normaloff>:/icons/thefile.svg</normaloff>:/icons/thefile.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="topWidget" native="true">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="leftCsdLayout">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item>
        <widget class="QToolButton" name="menuButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="thefile" resource="resources.qrc">
           <normaloff>:/icons/thefile.svg</normaloff>:/icons/thefile.svg</iconset>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="tabsLayout"/>
       </item>
       <item>
        <widget class="tWindowTabber" name="windowTabber" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="jobButtonLayout"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rightCsdLayout">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>1</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="tStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNewTab">
   <property name="icon">
    <iconset theme="tab-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionCloseTab">
   <property name="icon">
    <iconset theme="tab-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Close Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionShowHiddenFiles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Hidden Files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionGo">
   <property name="icon">
    <iconset theme="go-jump">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="edit-cut">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionMove_to_Trash">
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Move to Trash</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset theme="edit-select-all">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="icon">
    <iconset theme="window-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNew_Folder">
   <property name="icon">
    <iconset theme="folder-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">tstackedwidget.h</header>
   <container>1</container>
   <slots>
    <signal>switchingFrame(int)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>tWindowTabber</class>
   <extends>QWidget</extends>
   <header location="global">twindowtabber.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
